label_studio/io_storages/aperturedb/models.py

```diff
@@ -137,7 +125,7 @@ class ApertureDBImportStorageBase(ApertureDBStorageMixin, ImportStorage):
     )
 
     def iterkeys(self):
-        db = self.get_connection()
+        entry = time.time()
```

Comment:
unused variable?

Reply:
ah yes, that is unused now because of the pool. I was using it to keep track of time to get the connection started.